packages/react-reconciler/src/ReactFiberHooks.new.js

```diff
@@ -1783,6 +1786,47 @@ function updateEffect(
   return updateEffectImpl(PassiveEffect, HookPassive, create, deps);
 }
 
+function mountEvent<T>(callback: () => T): () => T {
+  const hook = mountWorkInProgressHook();
+  const ref = {current: callback};
+
+  function event(...args) {
```

Comment:
We don't use `...rest` syntax in this codebase because the transpiled output isn't great. Here's what it looks like for this function:

```js
  function event() {
    if (isAlreadyRenderingProd()) {
      throw new Error('An event from useEvent was called during render.');
    }

    for (var _len = arguments.length, args = new Array(_len), _key = 0; _key < _len; _key++) {
      args[_key] = arguments[_key];
    }

    return ref.current.apply(this, args);
  }
```

(Probably should update the repos linter to forbid this)

You can use old style `arguments` object instead